src/routes/safe/store/actions/createTransaction.js

```diff
@@ -57,16 +57,16 @@ const createTransaction = ({
   const from = userAccountSelector(state)
   const safeInstance = await getGnosisSafeInstanceAt(safeAddress)
   const lastTx = await getLastTx(safeAddress)
-  const nonce = await getNewTxNonce(txNonce, lastTx, safeInstance)
-  const isExecution = await shouldExecuteTransaction(safeInstance, nonce, lastTx)
+  const nonce = await getNewTxNonce(txNonce, lastTx, safeAddress)
+  const needsApproval = await doesTxNeedApproval(safeInstance, nonce, lastTx)
```

Comment:
why was this changed from checking if tx can be executed right way to checking if tx needs approval?

Reply:
the base branch is outdated, this change should not be here.

Reply:
this change does not appear anymore